mujoco_warp/_src/bvh.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2025 The Newton Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from typing import Tuple
+
+import warp as wp
+
+from .types import GeomType
+from .types import Model
+from .types import Data
+from .render_context import RenderContext
+
+wp.set_module_options({"enable_backward": False})
+
+
+@wp.func
+def compute_box_bounds(
```

Comment:
we can add an `_` prefix to this function and others in this module that are not part of the public api or will not be imported by other modules